### PR TITLE
feat: add a simple docker image for the webapp

### PR DESCRIPTION
**Pull request by bonomat**
I hope it works.... (tested on my fork)

https://github.com/bonomat/10101/pkgs/container/10101%2Fwebapp

### CODE REVIEW

File: webapp/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+ARG BINARY=target/release/webapp
+
+RUN apt-get update && \
+    apt-get install libpq5 -y && \
+    apt-get install sqlite3 -y && \
+    apt-get install ca-certificates openssl -y
+
+
+RUN mkdir cert
```
Comment (holzeis) [inline webapp/Dockerfile:14]: 🔧 i think the default path for the self signed certs is `certs`.

File: webapp/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+RUN cd cert && \
+    openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -sha256 -days 3650 -nodes -subj "/C=XX/ST=StateName/L=CityName/O=CompanyName/OU=CompanySectionName/CN=CommonNameOrHostname"
```
Comment (holzeis) [inline webapp/Dockerfile:17]: nit: we might want to to adapt the details just so that it doesn't look too generic?

File: webapp/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+USER 1000
+
+COPY $BINARY /usr/bin/webapp
+
+
+ENTRYPOINT ["/usr/bin/bash"]
```
Comment (holzeis) [inline webapp/Dockerfile:24]: 🔧

```suggestion
ENTRYPOINT ["/usr/bin/webapp"]
```

File: webapp/Dockerfile (hunk 2 above)
Comment (1010Tom) [reply webapp/Dockerfile:17]: I honestly don't know what to adapt here. I just took what I received from you 🙈

Comment (bonomat) [reply webapp/Dockerfile:17]: I honestly don't know what to adapt here. I just took what I received from you 🙈

**Review by 1010Tom — APPROVED** [review]
Looks good

**Review by holzeis — APPROVED** [review]

**Pull request merged**